src/helpers/roles.ts

```diff
@@ -0,0 +1,8 @@
+import { type APIInteractionGuildMember, GuildMember } from 'discord.js';
+
+export const isAdmin = (member: GuildMember | APIInteractionGuildMember | null): boolean =>
+  member instanceof GuildMember && member.roles.cache.some((role) => role.name === 'Admin');
+
+export const isModo = (member: GuildMember | APIInteractionGuildMember | null): boolean =>
+  member instanceof GuildMember &&
+  member.roles.cache.some((role) => role.name === 'Admin' || role.name === 'Modo');
```

Comment:
`GuildMember | APIInteractionGuildMember | null` is the type that you get from `interaction.member` and there is no .roles on `APIInteractionGuildMember` so I've abstracted that here to avoid check everywhere